### PR TITLE
LSR-935 Comment iOS code

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

File: ios/Framework/Core/Auth/SignUpScreenlet/SignUpViewModel.swift [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
 import UIKit
 
 
 @objc public protocol SignUpViewModel {
 
+	/// User email address.
```
Comment (nhpatt) [inline ios/Framework/Core/Auth/SignUpScreenlet/SignUpViewModel.swift]: I don't love this kind of comments but it's ok

File: ios/Framework/Core/Base/BaseListScreenlet/BaseListScreenlet.swift [hunk 2]
```diff
@@ -140,23 +159,50 @@ public class BaseListScreenlet: BaseScreenlet {
 	}
 	
 	
+	/// Prepares the page to be loaded in the list.
+	/// This method is intended to be overwritten by children classes.
```
Comment (nhpatt) [inline ios/Framework/Core/Base/BaseListScreenlet/BaseListScreenlet.swift]: overridden

File: ios/Framework/Core/Base/BaseScreenlet.swift [hunk 3]
```diff
@@ -302,10 +317,21 @@ import QuartzCore
...
+	///
+	/// - Parameters:
+	///   - name: action name.
+	///   - sender: source of the event.
+	/// - Returns: interactor.
 	public func createInteractor(name name: String, sender: AnyObject?) -> Interactor? {
 		return nil
 	}
 
+	/// endInteractor is invoked when an interaction should be ended.
```
Comment (nhpatt) [inline ios/Framework/Core/Base/BaseScreenlet.swift]: "when an interaction ends", ?

File: ios/Framework/Core/FileDisplay/FileDisplayScreenlet.swift [hunk 4]
```diff
@@ -119,7 +138,7 @@ public class FileDisplayScreenlet: BaseScreenlet {
 		return interactor
 	}
 
-	public func createLoadFileInteractor() -> Interactor? {
+	private func createLoadFileInteractor() -> Interactor? {
```
Comment (nhpatt) [inline ios/Framework/Core/FileDisplay/FileDisplayScreenlet.swift]: don't mix documentation with API changes :P

File: ios/Framework/Core/WebContent/DisplayScreenlet/WebContentDisplayScreenlet.swift [hunk 5]
```diff
@@ -1,27 +1,44 @@
...
 
 
 @objc public protocol WebContentDisplayScreenletDelegate : BaseScreenletDelegate {
 
+	///  Called when the web content’s HTML is received.
+	///
+	/// - Parameters:
+	///   - screenlet
+	///   - html: web content html.
+	/// - Returns: original or modify html.
```
Comment (nhpatt) [inline ios/Framework/Core/WebContent/DisplayScreenlet/WebContentDisplayScreenlet.swift]: modified

File: ios/Framework/Core/WebContent/DisplayScreenlet/WebContentDisplayViewModel.swift [hunk 6]
```diff
@@ -1,23 +1,25 @@
...
+ */
 import UIKit
 
 
 @objc public protocol WebContentDisplayViewModel {
 
+	/// WebContent html to be displayed.
 	var htmlContent: String? { get set }
 
+	/// DDLRecord receive to be displayed.
```
Comment (nhpatt) [inline ios/Framework/Core/WebContent/DisplayScreenlet/WebContentDisplayViewModel.swift]: ?

File: ios/Framework/Themes/Default/Auth/UserPortraitScreenlet/UserPortraitView_default.swift [hunk 7]
```diff
@@ -46,29 +46,39 @@ public class UserPortraitView_default: BaseScreenletView,
...
-		didSet {
-			portraitImage?.layer.borderColor = (borderColor ?? DefaultThemeBasicBlue).CGColor
-		}
-	}
+	@IBOutlet weak var editButton: UIButton?
+
 	override public var editable: Bool {
 		didSet {
-			self.editButton.hidden = !editable
+			self.editButton?.hidden = !editable
```
Comment (nhpatt) [inline ios/Framework/Themes/Default/Auth/UserPortraitScreenlet/UserPortraitView_default.swift]: :P

**Comment by nhpatt** [comment]
Ok, review the comments but LGTM 👍

File: ios/Framework/Core/Auth/SignUpScreenlet/SignUpViewModel.swift (hunk 1 above)
Comment (dgarciasarai) [inline ios/Framework/Core/Auth/SignUpScreenlet/SignUpViewModel.swift]: It's like comment a getter or setter, right? Should I remove them?

File: ios/Framework/Core/Base/BaseListScreenlet/BaseListScreenlet.swift (hunk 2 above)
Comment (dgarciasarai) [inline ios/Framework/Core/Base/BaseListScreenlet/BaseListScreenlet.swift]: 👍

File: ios/Framework/Core/Base/BaseScreenlet.swift (hunk 3 above)
Comment (dgarciasarai) [inline ios/Framework/Core/Base/BaseScreenlet.swift]: +1

File: ios/Framework/Core/FileDisplay/FileDisplayScreenlet.swift (hunk 4 above)
Comment (dgarciasarai) [inline ios/Framework/Core/FileDisplay/FileDisplayScreenlet.swift]: ok, sorry :/

File: ios/Framework/Core/WebContent/DisplayScreenlet/WebContentDisplayScreenlet.swift (hunk 5 above)
Comment (dgarciasarai) [inline ios/Framework/Core/WebContent/DisplayScreenlet/WebContentDisplayScreenlet.swift]: 👍

File: ios/Framework/Core/Auth/SignUpScreenlet/SignUpViewModel.swift (hunk 1 above)
Comment (jmnavarro) [inline ios/Framework/Core/Auth/SignUpScreenlet/SignUpViewModel.swift]: it the comment doesn't say anything more than the code itself, then it's pointless

**Comment by jmnavarro** [comment]
👍 
Anything pending can be done later

**Pull request merged**